content/blog/2021-01-21-unpack-cfr.Rmd

```diff
@@ -0,0 +1,249 @@
+---
+title: "Unpacking the Drop in COVID-19 Case Fatality Rates"
+author: "Helen Zhou, Cheng Cheng, Jeremy C. Weiss, Zachary C. Lipton"
+output:
+  bookdown::html_document2: default
+bibliography: refs.bib
+link-citations: yes
+---
+
+<!-- install.packages("bookdown") -->
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Introduction
+
+Since the COVID-19 pandemic first reached the United States, the case fatality (CFR) rate has fallen considerably. Between the first peak in mid-April and the second peak in mid-July, the case fatality rate fell from $7.9\%$ to the $0.7\%$--$2.3\%$ range, where it has since remained despite cases rising again into an (ongoing) third wave:
```

Comment:
the numbers here don't need to be in math mode

Reply:
Got it, fixed this and all similar cases.